plugins/venat-plugin-xiv-weather/src/data/weather.ts

```diff
@@ -0,0 +1,272 @@
+import { LocalizedAsyncResource } from './resource';
+import {
+  getTerritoryTypeIndex,
+  getWeatherIndex,
+  getWeatherRateIndex,
+  XIVAPILanguage,
+  XIVAPITerritoryType,
+  XIVAPIWeather,
+  XIVAPIWeatherRate,
+} from './xivapi';
+
+const SECONDS = 1;
+const MINUTES = 60 * SECONDS;
+const WEATHER_PERIOD = 23 * MINUTES + 20 * SECONDS;
+
+export enum Language {
```

Comment:
Maybe a separate issue to do later, but this enum is getting repeated in each module.

Reply:
Yeah, I considered moving it somewhere common, but it's implementation-specific to XIV.